contracts/interchain-token-service/tests/test.rs

```diff
@@ -175,3 +188,19 @@ fn transfer_ownership() {
 
     assert_eq!(client.owner(), new_owner);
 }
+
+#[test]
+fn interchain_token_deploy_salt() {
+    let (env, client) = setup_env();
+
+    let deployer: Address = Address::from_str(
+        &env,
+        "CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHK3M",
+    );
+
+    let salt = BytesN::<32>::from_array(&env, &[0; 32]);
```

Comment:
```suggestion
    let salt = BytesN::<32>::from_array(&env, &[1; 32]);
```
Maybe use a non zero value